nodebuilder/state/config.go

```diff
@@ -2,25 +2,21 @@ package state
 
 import (
 	"github.com/cosmos/cosmos-sdk/crypto/keyring"
-
-	"github.com/celestiaorg/celestia-node/state"
 )
 
 var defaultKeyringBackend = keyring.BackendTest
 
 // Config contains configuration parameters for constructing
 // the node's keyring signer.
 type Config struct {
-	KeyringAccName string
+	KeyringKeyName string
```

Comment:
We may add prefix it with Default to imply others can be used as well

Reply:
`DefaultKeyName`

Reply:
And `DefaultBackendName` for consistency